p3/templatetags/p3.py

```diff
@@ -654,3 +654,11 @@ def p3_voting_data(conference):
     results['groups'] = dict(groups)
     return results
 
+
+@fancy_tag(register, takes_context=True)
+def get_latest_conf_deadline(context, limit=None, not_expired=True):
+    try:
+        conf = ConferenceModels.Conference.objects.latest('code')
+    except IndexError:
+        conf = None
+    return [conf.name, conf.code, conf.conference_start, conf.conference_end, datetime.today().date()]
```

Comment:
It's better to move the return inside the try and return an empty array in the except clause, otherwise the return will raise an exception :)